torch/csrc/jit/passes/onnx/constant_map.cpp

```diff
@@ -304,6 +317,34 @@ void ConstantValueMap::PrintMaps() {
       std::cout << std::endl;
     }
   }
+  std::cout << std::endl;
+  std::cout << "InferredShape Map:" << std::endl;
+  count = 0;
+  for (const auto& x : ConstantValueMap::getInstance().inferredShapeData) {
+    std::cout << "(node " << x.first << ": ";
```

Comment:
remove debug prints?

Reply:
I thought it was required to add them here while introducing a new map so I am totally fine to remove them. To confirm, do you want me only remove my updates or the whole `PrintMaps`?

Reply:
Oh I'm sorry I misunderstood. Please keep it!